zeebe/protocol-impl/src/test/java/io/camunda/zeebe/protocol/impl/JsonSerializableToJsonTest.java

```diff
@@ -2460,6 +2461,24 @@ final var record =
         }
         """
       },
+      {
```

Comment:
```suggestion
      /////////////////////////////////////////////////////////////////////////////////////////////
       ///////////////////////////////// UserRecord //////////////////////////////////////
       /////////////////////////////////////////////////////////////////////////////////////////////
      {
```

🔧 Let's add the "header" that we do for all other scenarios as well.